SbtPlugin/src/main/scala/com/dslplatform/sbt/SbtDslPlatformPlugin.scala

```diff
@@ -36,6 +36,7 @@ object SbtDslPlatformPlugin extends AutoPlugin {
     val dslLatest = settingKey[Boolean]("Check for latest versions (dsl-compiler, libraries, etc...)")
     val dslForce = settingKey[Boolean]("Force actions without prompt (destructive migrations, missing folders, etc...)")
     val dslPlugins = settingKey[Option[File]]("Path to additional DSL plugins")
+    val dslCompilerURL = settingKey[Option[String]]("Download URL for a custom DSL compiler")

```

Comment:
better name would be dslDownload
since other stuff are downloaded from there also
Ideally later we could use network path alongside URL